src/site/content/en/blog/variable-fonts/index.md

```diff
@@ -8,360 +8,710 @@ authors:
   - mustafakurtuldu
   - thomassteiner
 date: 2018-02-19
-updated: 2020-05-12
+updated: 2020-08-05
 hero: hero.jpg
 tags:
   - blog
   - fonts
 ---
+<!--
 
-Variable fonts can significantly reduce the size of your font files and make
-it possible to animate your font characters. This guide explains how
-variable fonts work, how typographers implement variable fonts, and how
-to work with variable fonts in CSS.
+TODO add these to "authors" in yaml ↑
+{ % include "web/_shared/contributors/dcrossland.html" %}
+{ % include "web/_shared/contributors/roeln.html" %}
 
-## Browser compatibility {: #compatibility }
+-->
 
-As of May 2020 variable fonts are supported in most browsers. See
-[Can I use variable fonts?](https://caniuse.com/#feat=variable-fonts)
-and [Fallbacks](#fallbacks).
+In this article, we will look at what variable fonts are, the benefits they
+offer, and how we can use them in our work. First, let's review how typography
+works on the web, and what innovations variable fonts bring.
```

Comment:
Please put the browser compatibility data back in. We know from the [MDN DNA survey](https://mdn-web-dna.s3-us-west-2.amazonaws.com/MDN-Web-DNA-Report-2019.pdf) (page 19) that browser compatibility is a top concern for web developers. We put it at the top of articles to convince developers that the rest of the content is worth their time (it's very frustrating to read a guide only to find out at the end that an API is only supported in Chrome).